extension_explorer/templates/publishers.html

```diff
@@ -175,7 +175,7 @@ <h3>
         </h3>
 
         <p>
-          {{ _("You will need to find somewhere to host your extension online at a stable URL. Creating a GitHub repository for your extension is a good option because it offers version control and issue tracking.") }}
+          {{ _("You will need to find somewhere to host your extension online at a stable URL. It must be possible to access an extension's schema and codelist files by replacing extension.json in the extension's URL with a file's path, e.g. <code>release-schema.json</code> or <code>codelists/codelist.csv</code>. A permissible alternative is to use an API that behaves like a directory. Creating a GitHub repository for your extension is a good option because it offers version control, issue tracking and provides the appropriate file URLs.") }}
```

Comment:
I cut "A permissible alternative is to use an API that behaves like a directory. " as we don't discuss directories in this paragraph. If the suggestion is accepted, please assign to @yolile  for [translation](https://github.com/open-contracting/extension-explorer?tab=readme-ov-file#translations).

```suggestion
          {{ _("You will need to find somewhere to host your extension online at a stable URL. It must be possible to access an extension's schema and codelist files by replacing <code>extension.json</code> in the extension's URL with a file's path: for example, <code>release-schema.json</code> or <code>codelists/codelist.csv</code>. Creating a GitHub repository for your extension is a good option because it offers version control, issue tracking and appropriate URLs: for example, <code>https://raw.githubusercontent.com/open-contracting-extensions/ocds_bid_extension/master/extension.json</code>.") }}
```

Reply:
that makes sense, the place that discusses the directory is in the extension-template so this piece of guidance can be incorporated when the template guidance is moved from it's README to here